src/Apps/Debug/DebugApp.tsx

```diff
@@ -8,6 +12,46 @@ export const DebugApp: React.FC<{}> = () => {
       <Title>Baseline</Title>
 
       <Text variant="sm-display">Baseline</Text>
+      {/* TODO: REMOVE THIS BEFORE MERGE */}
```

Comment:
for this file we can just checkout the `DebugApp.tsx` app from main, but it was a great TIL